### DISCUSSION TITLE
Ask some questions

### DISCUSSION THREAD

**Discussion by hyjklmn** [saadeghi/daisyui#2250] [General]
When i fork this repo and first open svelte file,i see this error?

![Snipaste_2023-08-15_18-59-06](https://github.com/saadeghi/daisyui/assets/73629147/d954854c-0e34-43ee-b4d7-ee6c1d3f1a30)

**Comment by saadeghi** [comment]
I see that too but there's no issues neither in dev environment or production.

The error is coming from the `mdsvex` plugin (markdown for Svelte) but everything works regardless. The config is also correct according to the `mdsvex` docs so I assume it's a bug from `mdsvex`

You can safely ignore that error.

I will search on `mdsvex` GitHub to see if the issue is addressed or not.

**Reply by saadeghi** [reply]
This one is not error. Not everything in terminal is an error. It's an echo command that is needed for running the dev server for the first time (not reloads)

It is needed because Vite is unable to load CSS files conditionally for different environments and it won't re-process the styles on demand. So for the first time you run the dev server, the dev environment styles should be handled using a separate component file which is ignored from git. This was the only efficient way I found to handle the correct CSS files for both production and development environment without an extra node dependency.
